skimage/measure/_regionprops.py

```diff
@@ -108,6 +108,11 @@ class _RegionProperties(object):
 
     def __init__(self, slice, label, label_image, intensity_image,
                  cache_active):
+                 
+        if not intensity_image is None:
```

Comment:
Python, perhaps somewhat oddly but very usefully, allows us to write:

```
```if intensity_image is not None